cost-optimization-aks-content.md

```diff
@@ -0,0 +1,795 @@
+In this document, we'll discuss features and options, and the best practices for running and managing Kubernetes workloads on AKS in a cost optimized way to take advantage of the scalability provided by Azure. 
+
+> [!NOTE]
+> This document assumes you are familiar with Kubernetes, Azure, AKS and autoscaling. If you need a refresher, see the [**Related articles**](#related-articles) section for resources.
+
+## Introduction
+
+As Kubernetes gains widespread adoption, a growing number of enterprises and platform-as-a-service (PaaS) and software-as-a-service (SaaS) providers are using multi-tenant Kubernetes clusters for their workloads. This means that a single cluster might be running applications that belong to different teams, departments, customers, or environments. The multi-tenancy provided by Kubernetes lets companies manage a few large clusters, instead of multiple smaller ones, with benefits such as appropriate resource utilization, simplified management control, and reduced fragmentation.
+
+Over time, some of these companies with fast-growing Kubernetes clusters start to experience a disproportionate increase in cost. This happens because traditional companies that embrace cloud-based solutions like Kubernetes don't have developers and operators with cloud expertise. This lack of cloud readiness leads to applications becoming unstable during autoscaling (for example, traffic volatility during a regular period of the day), sudden bursts, or spikes (such as TV commercials or peak scale events like Black Friday and Cyber Monday). In an attempt to "fix" the problem, these companies tend to over-provision their clusters the way they used to in a non-elastic environment. Over-provisioning results in considerably higher CPU and memory allocation than what applications use for most of the day.
```

Comment:
> This happens because traditional companies that embrace cloud-based solutions like Kubernetes don't have developers and operators with cloud expertise.

I'd try to find another phase here.  This is pretty harsh to say.  You can be a company that experiences this problem even if you have cloud expertise.  And cloud expertise isn't an "on" or "off" state, so hard to say when you have it or not.  The premise of "over-provision to fix scaling issues" is a good angle to lead with in this article, I'd suggest doing it though with less direct "attack" on the customer's capabilities, experiences, or staff.